### DISCUSSION TITLE
Source code repo separated from locales repo

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#5910] [Q&A] [answered]
I would like to set up a Django-based application for translation with a separate repository for the locales.

Is there documentation or an example on how to set up a separate repository for locales? (I didn't find it in the Weblate document)

**Comment by nijel** [comment, marked as answer]
It is there ;-). See https://docs.weblate.org/en/latest/faq.html#how-can-i-limit-weblate-access-to-only-translations-without-exposing-source-code-to-it

**Reply by rffontenelle** [reply]
Thanks!
